org.bndtools.launch/src/bndtools/launch/AbstractOSGiLaunchDelegate.java

```diff
@@ -294,6 +296,13 @@ public String[][] getClasspathAndModulepath(ILaunchConfiguration config) throws
 			classpathAndModulepath = new String[2][];
 			classpathAndModulepath[1] = new String[0];
 		}
+		if (classpathAndModulepath[1].length > 0) {
+			classpathAndModulepath[1] = Arrays.asList(classpathAndModulepath[1])
+				.stream()
```

Comment:
```suggestion
			classpathAndModulepath[1] = Arrays.stream(classpathAndModulepath[1])
```

You can stream an array directly without needing to wrap in a list.